libraries/RC_Channel/RC_Channel.h

```diff
@@ -614,6 +618,7 @@ class RC_Channels {
     uint32_t last_update_ms;
     bool has_new_overrides;
     bool _has_had_rc_receiver; // true if we have had a direct detach RC reciever, does not include overrides
+    bool _had_had_override; // true if we have had a override on any channel
```

Comment:
sorry to be annoying but this should probably be, "_ha**s**_had_override"?

Reply:
fixed